Develop isdm gmm

glynm opened the pull request.

glynm committed:
more development on RoomBookingV06 (Unicode) - in specific the reporting of meeting duration and remaining time.

glynm committed:
fixing the diagnostics page location name field, so the text displays as it should.

thatgitsam commented:
There are already debug constants located in the Debug include, and would prefer not to debug using compiler directives. This file is spread across pretty much all modules,. Would it make more sense for you to reference a local copy of it instead?

glynm committed:
1. putting the CustomFunctions.axi back as it was.
2. updating the scShedulingRoomInstance.axs module:
   a. adding a local function for fnSecondsToDurationTextLocal().
   b. adding in a configuration switch for disabling the Ending Meeting message popup.

glynm commented:
I have cleaned up the CustomFunctions.axi - and used yoursuggestion, and made a local function in the scScheduliingRoomInstance module instead.

glynm committed:
Merge branch 'master' into develop_ISDM_GMM
